include/capstone.h

```diff
@@ -27,19 +27,23 @@ extern "C" {
 #pragma warning(disable:4201)
 #pragma warning(disable:4100)
 #define CAPSTONE_API __cdecl
+#ifndef CAPSTONE_EXPORT
```

Comment:
CAPSTONE_EXPORT is only defined in the next few lines, so but you check it here. what is your intention?